### ISSUE TITLE
Scaler not scaling all dimensions

### ISSUE THREAD

**Issue by FIrgolitsch** [ome/ome-zarr-py#262]
When using writer.image_write the default "nearest" scaling method only scales the x and y axes (when given zxy axes). Gaussian and Laplacian scale the axes, but as mentioned in Issue #254 other methods than "nearest" aren't working. Is there any way to specify which axes to scale? I tried passing coordinate transforms to the method but that didn't work either.

**Comment by will-moore** [comment]
Thanks for the feedback.
I'm afraid the non-scaling of Z is due to the fact that historically, many OME and similar tools have considered "big" pyramid images to be a stack of 2D planes, rather than a 3D volume, so the number of Z planes is the same at all resolutions.
E.g. bioformats2raw generates pyramids not downsampled in Z, OMERO API and viewers too, and for OME-NGFF viewing in e.g. [vizarr](https://github.com/hms-dbmi/vizarr/pull/71)

Some users may want to preserve this behaviour while others will certainly want nD scaling.

There are other scaling issues - I think #254 is the other issue you mentioned, so we need to properly review the scaling API.

**Comment by will-moore** [comment]
No, update I'm afraid, but I'm aware it's something that needs addressing, so thanks for the ping.
